### ISSUE TITLE
'namespace' rule doesn't check jsx

### ISSUE THREAD

**Issue by jf248** [benmosher/eslint-plugin-import#1150]
The `namespace` rule doesn't raise an error for:
```
import * as MyComponents from './my-components'
const Foo = <MyComponents.DoesNotExist/>
```
I think it would be helpful if it did.

(It doesn't because here `MyComponents.DoesNotExist` is a `JSXMemberExpression` which the rule doesn't check.)

Submitting a PR to 'fix' this.
